CHANGES.md

```diff
@@ -1,5 +1,12 @@
 ## Version 2.1.4 (in development)
-
+* Included normalization of time in case it is called 't' in a dataset instead of 'time' (#956). 
+  This is the case for the following datasets: 
```

Comment:
I don't like listing all the affected dsr id's. You already list them in the linked issue, that should be enough. But let's see what is Norman's opinion on this.